### DISCUSSION TITLE
Dark mode not detected on capacitor/android

### DISCUSSION THREAD

**Discussion by wieli99** [quasarframework/quasar#9201] [CLI - Capacitor / Cordova modes]
Hi everyone. I'm looking for a way to support dark mode on capacitor apps on android. 
`$q.dark.mode` is set to "auto", however the app does not recognize the system dark mode -> `$q.dark.isActive` returns false 

Does anyone know how to correctly respect the system prefs for this?

It works perfectly fine in the browser (both on desktop and android)

**Comment by ontwikkelfabriek** [comment]
Here's an article about why it's not working and what you can do about it:
https://medium.com/@hinddeep.purohit007/supporting-dark-mode-in-ionic-capacitor-8e57fe9dbd47
